static/docs/use-cases/data-registry.md

```diff
@@ -0,0 +1,113 @@
+# Data Registry
```

Comment:
should stay in a separate PR?

Reply:
Oh shoot. That shouldn't be here... Let me check what happened.

Reply:
Ok removed. That was an error, thanks for noticing!